src/components/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./HeroSection.scss";
+import HeroImage from "./assets/hero-image.svg";
+
+function HeroSection() {
+	return (
+		<div className="HeroSection">
+			<div className="HeroSection__info">
+				<h1 className="HeroSection__title">
+					Uniting <span className="HeroSection__title--circled">skill</span>{" "}
+					with opportunity
+				</h1>
+				<p className="HeroSection__text">
+					SkillChain connects job seekers with top employers to find the perfect
+					match for job vacancies.
+				</p>
+				<button className="HeroSection__button" onClick={()=> window.open('https://www.google.com', '_blank')}>View Jobs</button>
```

Comment:
Would be better to just use an a-tag here :D Sorry for confusing!

Reply:
you mean, return to the previous state? like it was?

Reply:
`<a className="HeroSection__button" href="https://www.google.com" target="_blank">View Jobs</a>`

Reply:
done!